themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -25,7 +25,7 @@
   <form class="form-inline">
 
     <div class="full-facet-sort-options form-group" role="group" aria-labelledby="full-facet-sort-label">
-      <label id="full-facet-sort-label" class="control-label" for="btn-group-sort"><?=$this->translate('Sort') ?></label>
```

Comment:
The for attribute cannot be used to refer to an element that's not a form control. Since we have two controls, it's better not to refer to either.